package/gluon-mesh-batman-adv-core/files/lib/gluon/upgrade/310-gluon-mesh-batman-adv-core-mesh

```diff
@@ -3,6 +3,7 @@
 local sysconfig = require 'gluon.sysconfig'
 local sysctl = require 'gluon.sysctl'
 local site = require 'gluon.site_config'
+local util = require 'luci.util'
```

Comment:
I guess we shouldn't add new LuCI dependencies when we're elsewhere trying to get rid of it...